packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -566,6 +566,81 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
     }).then<int>((_) => count);
   }
 
+  /// Triggers multiple frames for `duration` amount of time or until there are
+  /// no longer any frames scheduled, whichever is earlier.
+  ///
+  /// This does not promise to exhaust all pending frames. There maybe remaining
+  /// animation frames and therefore undisposed `Ticker`s.
+  ///
+  /// This will call [pump] at least once, even if no frames are scheduled when
+  /// the function is called, to flush any pending microtasks which may
+  /// themselves schedule a frame.
+  ///
+  /// The `frameRefreshRate` specify the frequency (in Hz) for frame requests.
+  /// This is an analog to screen refresh rate on a device. The frequency rate
+  /// will be rounded down to integer microseconds frame interval.
+  /// Default value is 59.94 Hz, which is the real refresh rate for many devices
+  /// when it shows as 60 Hz, for historical reason.
```

Comment:
we should just use 60Hz, which is what we use everywhere else. It doesn't matter what the actual device rate is since this is artificial.

...except maybe you mean this to not be artificial? I admit I don't really understand the use case here.

Reply:
I used 59.94Hz in pumpFrames first in order to simulate the real devices. @CareF feel free to change pumpFrames if we agree to unify to 60. There shouldn’t break anything since the difference is so small.

Reply:
> we should just use 60Hz, which is what we use everywhere else. It doesn't matter what the actual device rate is since this is artificial.
> 
> ...except maybe you mean this to not be artificial? I admit I don't really understand the use case here.

I believe there is nothing more than a number. The historical reason I learnt is to avoid resonance with 60Hz city electricity for old TVs and than it is used for newer screens as a convention (but not always respected). I'll change it to 60Hz.

Reply:
@dkwingsmt It seems that change `pumpFrames` to 60Hz introduces a Golden file change.